client/layout/guided-tours/docs/API.md

```diff
@@ -0,0 +1,223 @@
+* `when`: (function, optional) This is a redux selector that can prevent step from showing when it evaluates to false. Define `next` prop to tell Guided Tours name of the step it should skip to. If you omit this prop, step will be rendered as expected.
+* `next`: (string, optional) Define this to tell Guided Tours name of the step it should skip to when `when` evaluates to false.
+* `children`: (nodes) Content of step. Usually a paragraph of instructions and some controls for tour. See below for available options.
+
+### Example
+
+```jsx
+// single step tour
+<Tour path="/me" name="uselessTour">
+  <Step>
+    <p>This is a page about you.</p>
+    <ButtonRow>
+      <Quit />
+    </ButtonRow>
+  </Step>
+</Tour>
+```
+
+Note that all text content needs to be wrapped in `<p>` so it gets proper styling.
+
+For more comprehensive examples, look at [TUTORIAL.md](TUTORIAL.md) or explore existing tours in client/layout/guided-tours/tours.
+
+## ButtonRow
+
+ButtonRow is a React component to display button controls in Step and takes care of their proper styling. Usually used as the last child of Step to contain all available controls.
+
+### Example
+
+```jsx
+<Step>
+  <p>ButtonRow Example</p>
+  <ButtonRow>
+    <Next step="next-step" />
+    <Quit />
+  </ButtonRow>
+</Step>
+```
+
+## Continue
+
+Continue is a React component that you can use in Step to programmatically continue the tour to other step based on user interaction with Calypso.
+
+### Props
+
+* `step`: (string) Name of the step the tour will advance to.
+* `target`: (string, optional) Name of `[data-tip-target]` that would be watched.
+* `click`: (bool, optional) If true, `onClick` will be listened on `target` DOM node.
+* `hidden`: (bool, optional) If true, this will not render anything in Step, while functionality remains.
+* `icon`: (string, optional) Name of Gridicon to show in custom message.
+* `when`: (function, optional) Redux selector. Once it evaluates to true, tour will advance to `step`.
+* `children`: (nodes, optional) If you provide children, it will override the default content.
+
+### Visual Options
+
+There are three ways to define a content of this component.
+
+- default text is "Click to continue." Use just as `<Continue … />` with no children.
+- default text + Gridicon ("Click *icon* to continue.") - provide name of Gridicon as an `icon` prop. Example: `<Continue icon="my-sites" … />`
+- custom content - provide it as children: `<Continue …>My Content</Continue>`
+
+### Functionality Options
+
+There are currently two ways to declare the condition to continue the tour.
+
+- Binding an `onClick` listener to a DOM node marked as `[data-tip-target]`
+- Redux selector function that evaluates to true in order to advance the tour
```

Comment:
"There are currently two ways to declare the condition to continue the tour" is a more general statement than "here's how you use Continue", which is why I'd add the Next button (with mentioning the caveat that it's not for use with Continue, but an alternative) — or reword the "two ways" sentence.